dist/api_data.py

```diff
@@ -1198,18 +1198,22 @@ def __cmp__(self, other):
 'WT_SESSION.prepare_transaction' : Method([
     Config('prepare_timestamp', '', r'''
         set the prepare timestamp for the updates of the current transaction.
-        The supplied value should not be older than any active read timestamps.
+        The supplied value must not be older than any active read timestamps.
         This configuration option is mandatory.  See
         @ref transaction_timestamps'''),
 ]),
 
 'WT_SESSION.timestamp_transaction' : Method([
     Config('commit_timestamp', '', r'''
         set the commit timestamp for the current transaction.  The supplied
-        value should not be older than the first commit timestamp set for the
-        current transaction.  The value should also not be older than the
+        value must not be older than the first commit timestamp set for the
+        current transaction.  The value must also not be older than the
         current oldest and stable timestamps.  See
         @ref transaction_timestamps'''),
+    Config('read_timestamp', '', r'''
```

Comment:
Why we only accept "read_timestamp" but not "round_to_oldest" in set_timestamp ?

Reply:
I took a look at this PR. The code appears to accept `round_to_oldest` but it is not part of the API definition above. That should be fixed.